Makefile

```diff
@@ -197,17 +199,17 @@ ifneq ($(shell docker images -q $(VELA_CORE_TEST_IMAGE)),)
 	docker rmi -f $(VELA_CORE_TEST_IMAGE)
 endif
 
+end-e2e:
+	sh ./hack/e2e/end_e2e.sh
+
+
 # load docker image to the kind cluster
 kind-load:
-	docker build -t $(VELA_CORE_TEST_IMAGE) .
+	docker build -t $(VELA_CORE_TEST_IMAGE) --build-arg=is_e2e=true .
 	kind load docker-image $(VELA_CORE_TEST_IMAGE) || { echo >&2 "kind not installed or error loading image: $(VELA_CORE_TEST_IMAGE)"; exit 1; }
 
-# Run tests
-core-test: fmt vet manifests
```

Comment:
why deleted?

Reply:
I thought this test isn't called by anywhere? maybe called manully？

Reply:
oh, yes, it could be deleted